css/index.css

```diff
@@ -298,6 +298,7 @@ tr:nth-child(even) {
 
 .myClass {
     display: none;
+    height: 150px;
```

Comment:
use `min-height` instead, that way it'll always be a minimum of `150px`, but will expand if the height is greater